org.rdkit.knime.types/rdkit-chemsrc/org/rdkit/knime/types/RDKitMolValueRenderer.java

```diff
@@ -382,12 +379,72 @@ else if (m_svgDocument != null) {
 			g.setColor(Color.black);
 		}
 	}
+	
+	@Override
+	public LockedSupplier<SVGDocument> getSvgSupplier() {
+      return new LockedSupplier<SVGDocument>(getSvg(), m_reentrantLock);
+	}
 
 	/**
 	 * {@inheritDoc}
 	 */
 	@Override
-	public SVGDocument getSvg() {
+	public synchronized SVGDocument getSvg() {
+		if (m_molecule != null) {
+			try {
+				final MolDraw2DSVG molDrawing = new MolDraw2DSVG(-1, -1);
+				
+				// Apply config from preferences (since RDKit Types version 4.6.0), if available
+				String strJsonConfig = RDKitDepicterPreferencePage.getJsonConfig();
+				if (strJsonConfig != null) {
+					RDKFuncs.updateDrawerParamsFromJSON(molDrawing, strJsonConfig);
+				}
+				
+				MolDrawOptions opts = molDrawing.drawOptions();
+				
+				// We've already prepared the molecule appropriately, so don't try again
+				opts.setPrepareMolsBeforeDrawing(false);
+				
+				// Apply old config, if no JSON config is provided (before RDKit Types version 4.6.0)
+				if (strJsonConfig == null) {
```

Comment:
Please note that the JSON file that I sent you yesterday does not do anything about stereo annotation.

Reply:
Done, added stereo annotation also in new default settings, increased a bit the bond line width to make Greg happy :-).